code-challenges.test.js

```diff
@@ -13,34 +13,109 @@
 
 // a) Create a test with an expect statement using the variables provided.
-
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
+describe("threeRemainder", () => {
+  it("returns an array of only the REMAINDERS of the numbers when divided by 3", () => {
+      const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+      // Expected output: [ 2, 0, -1, 0 ]
+    expect(threeRemainder(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+      const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+      // Expected output: [ 2, 1, -1 ]
+    expect(threeRemainder(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
+// ReferenceError: threeRemainder is not defined
 
 // b) Create the function that makes the test pass.
+// pseudocode
+// create a function called threeRemainder
+// input is an array of mixed data
+// output is an array of only the remainder of the numbers divided by 3
+// take the array of mixed data and iterate over it filtering out only the data types that are numbers - set that to a variable
+// map over the filter array and get the modulo of each value (value % 3)
+// return that array to receive an array of only the remainder of those numbers divided by 3 
+const threeRemainder = (arrayMixedData) => {
+  numArray = arrayMixedData.filter(value => typeof value == "number")
+  return numArray.map(value => value % 3)
+}
+// Test Suites: 1 passed, 1 total
+// Tests:       2 passed, 2 total
+
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
-
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+describe("cubedSum", () => {
+  it("returns the sum of all the numbers cubed", () => {
+      const cubeAndSum1 = [2, 3, 4]
+      // Expected output: 99
+    expect(cubedSum(cubeAndSum1)).toEqual(99)
+     const cubeAndSum2 = [0, 5, 10]
+      // Expected output: 1125
+    expect(cubedSum(cubeAndSum2)).toEqual(1125)
+  })
+})
+// ReferenceError: cubedSum is not defined
 
 // b) Create the function that makes the test pass.
+// pseudocode
+// create a function called cubedSum
+// input is an array of numbers
+// output is the sum of all the numbers cubed
+// iterate over the array using .map() and cube each number - set that to a variable
+// declare another variable that starts at 0 - this will be the start of the running sum
+// iterate over the mapped cubed array using a for loop and add each value of the array to the 0 variable so we can keep a running count
+// return the total of the cubed numbers
+
+const cubedSum = (array) => {
+  cubedArray = array.map(value => value **3)
+  let runningSum = 0
+  for(let i = 0; i < cubedArray.length; i++){
```

Comment:
Awesome work!  A possible refactor to consider would be to use a higher order function like reduce that will allow you to not only access the current value, but will also allow you to have a running sum in it as well.  Definitely a higher order function to dig into at some point.